blockprotocol/blockprotocol
# [SITE] Add Google Tag Manager init

Adds GTM to the site.

### To test
1. Install the [Google Analytics debugger extension](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna/related) - alternatively, watch the network tab for requests
2. Remove the 'VERCEL_ENV' check
3. Switch pages, check pageview events are fired
4. Will need to test again once merged
